roles/confluent.ksql/defaults/main.yml

```diff
@@ -26,6 +26,18 @@ ksql_default_internal_replication_factor: "{{ [ groups['kafka_broker'] | length,
 
 ksql_service_id: default_
 
+ksql_metrics:
+  enabled: true
+  consumer_interceptors: []
+  producer_interceptors: []
+  listener: 
```

Comment:
i wonder if you could set this listener = kafka_broker.listeners.external? and i guess move servers field up in the dictionary
Same goes for the other dictionaries

Reply:
Didn't want to do that, as the kafka_broker `listener` collection doesn't provided the `servers` field. Which is part of the goal of this feature, to support clusters outside of the running playbook.

Reply:
ya i get that, just make it
```
ksql_metrics:
  enabled: true
  consumer_interceptors: []
  producer_interceptors: []
  servers: "{{groups['kafka_broker']}}"
  listener: "{{kafka_broker_listeners[ksql_kafka_listener_name]}}
```